scripts/install_mason.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+git submodule update --init .mason
+
+export PATH="`pwd`/.mason:${PATH}"
+export MASON_DIR="`pwd`/.mason"
+
+mason install boost 1.58.0
```

Comment:
Possible to capture these version numbers in variables somewhere rather than needing to repeat them manually in `binding.gyp`?